spec/lib/declare_schema/migration_generator_spec.rb

```diff
@@ -2461,13 +2425,13 @@ class Advert < active_record_base_class.constantize
         migration_content = File.read(migrations.first)
         first_line = migration_content.split("\n").first
         base_class = first_line.split(' < ').last
-        expect(base_class).to eq("(ActiveSupport::VERSION::MAJOR >= 5 ? ActiveRecord::Migration[4.2] : ActiveRecord::Migration)")
+        expect(base_class).to eq("(ActiveRecord::Migration[4.2])")
       end
     end
 
     context 'Does not generate migrations' do
       it 'for aliased fields bigint -> integer limit 8' do
-        if ActiveSupport::VERSION::MAJOR >= 5 || !ActiveRecord::Base.connection.class.name.match?(/SQLite3Adapter/)
+        if !ActiveRecord::Base.connection.class.name.match?(/SQLite3Adapter/)
```

Comment:
I think we should delete this `if` condition since the deleted portion `ActiveSupport::VERSION::MAJOR >= 5 ||` would always evaluate to true.